src/subscription_manager/managercli.py

```diff
@@ -371,7 +371,7 @@ def main(self, args=None):
 
         # TODO: For now, we disable the CLI entirely. We may want to allow some commands in the future.
         if rhsm.config.in_container():
-            system_exit(os.EX_CONFIG, _("subscription-manager is disabled when running inside a container. Please refer to your host system for subscription management.\n"))
+            system_exit(os.EX_CONFIG, _("%s is disabled when running inside a container. Please refer to your host system for subscription management.\n") % "subscription-manager")
```

Comment:
Instead of a string literal, make it a "constant"

ie,
```
NAME="subscription-manager"
```

(if we had a constant.py, that would be a good place, but managercli.py is probably fine.)